diff --git a/dist/cjs/classes/job.js b/dist/cjs/classes/job.js
index 31bc35cfeba84d59db0aa7722f5f9c0b0dfaf9eb..d67dee450c960590f1b631efc6a6375cb69ac1bc 100644
--- a/dist/cjs/classes/job.js
+++ b/dist/cjs/classes/job.js
@@ -14,6 +14,7 @@ const optsDecodeMap = {
     kl: 'keepLogs',
     rdof: 'removeDependencyOnFailure',
 };
+BigInt.prototype.toJSON = function() { return this.toString() };
 const optsEncodeMap = (0, lodash_1.invert)(optsDecodeMap);
 exports.PRIORITY_LIMIT = 2 ** 21;
 /**
